models/public/hrnetv2/hrnetv2.md

```diff
@@ -0,0 +1,110 @@
+# hrnetv2
+
+## Use Case and High-Level Description
+HRNetV2-W48 is semantic-segmentation model based on architecture described in paper
+[High-Resolution Representations for Labeling Pixels and Regions](https://arxiv.org/abs/1904.04514).
+This is PyTorch implementation based on retaining high resolution representations throughout the model
+and pretrained on ADE20k dataset. The model input is a blob that consists of a
+single image of "1x3x320x320" in RGB order.
```

Comment:
you have input specification below, no need to include it into common description
```suggestion
and pretrained on ADE20k dataset. 
```